### DISCUSSION TITLE
Ability to have Telemetry & RCM update and be synced when Viewing Replays

### DISCUSSION THREAD

**Discussion by TheBaconCat** [f1multiviewer/issue-tracker#83] [Feature requests] [answered]
Currently when attempting to watch a replay the telemetry data does load but it seems to be pulling the last data that is had from the session and does not update at all during the replay. It would be nice to have the telemetry as well as RCM be synced to the replays for the complete immersive experience!

![image](https://user-images.githubusercontent.com/11319293/176981744-d24fe408-db07-4731-a9f1-0428e424d096.png)

**Comment by pvanb** [comment]
I really hope this is possible. I rarely watch races live so I've never been able to use this feature, which looks fantastic.

**Comment by f1multiviewer** [comment, marked as answer]
This has been possible for some time now!
